frontend/javascripts/oxalis/model/accessors/dataset_accessor.ts

```diff
@@ -321,11 +323,11 @@ export function getDatasetExtentAsString(
 
   if (inVoxel) {
     const extentInVoxel = getDatasetExtentInVoxel(dataset);
-    return `${formatExtentWithLength(extentInVoxel, (x) => `${x}`)} voxel`;
+    return `${formatExtentInUnitWithLength(extentInVoxel, "vx", (x, _) => `${x}`)} voxel`;
   }
 
-  const extent = getDatasetExtentInLength(dataset);
-  return formatExtentWithLength(extent, formatNumberToLength);
+  const extent = getDatasetExtentInUnit(dataset);
+  return formatExtentInUnitWithLength(extent, dataset.dataSource.scale.unit, formatNumberToLength);
```

Comment:
according to my suggestion from above, this would be:

```
  return formatExtentInUnitWithLength(extent, (length) => formatNumberToLength(length, dataset.dataSource.scale.unit));
```